Tests/TensorFlowTests/SequentialTests.swift

```diff
@@ -48,7 +48,6 @@ final class SequentialTests: XCTestCase {
             rmsprop.update(&model, along: 𝛁model)
             rmsprop.update(&model.allDifferentiableVariables, along: 𝛁model)
             adam.update(&model, along: 𝛁model)
-            adam.update(&model.allDifferentiableVariables, along: 𝛁model)
```

Comment:
Let's keep the `allDifferentiableVariable` version for now for source compatibility. We'll remove them in `0.4.1` along with `AllDifferentiableVariables`.

Reply:
Sounds good. I duplicated the implementation in this case because I can't have it call the new update function and we can just remove the duplicate code then.